bouncerscript/tests/test_script.py

```diff
@@ -194,6 +217,13 @@ async def toggled_get_locations_info(*args, **kwargs):
             mocker.patch.object(bscript, "get_locations_info", new=mock)
         await bouncer_locations(locations_context)
 
+        # We don't run this check unless info is being updated.
+        if updated_info:
+            if require_successive_versions and updated_info:
```

Comment:
```suggestion
            if require_successive_versions:
```
We already know `updated_info` is true-ish here.

Reply:
Whoops - thanks. I meant to undo that before pushing, heh.